packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -317,6 +318,79 @@ curl -v -X POST \
 ##### Response Example
 
 
+```json
+{
+  "id": "00ub0oNGTSWTBKOLGLNR",
+  "status": "ACTIVE",
+  "created": "2013-07-02T21:36:25.344Z",
+  "activated": null,
+  "statusChanged": null,
+  "lastLogin": null,
+  "lastUpdated": "2013-07-02T21:36:25.344Z",
+  "passwordChanged": "2013-07-02T21:36:25.344Z",
+  "profile": {
+    "firstName": "Isaac",
+    "lastName": "Brock",
+    "email": "isaac.brock@example.com",
+    "login": "isaac.brock@example.com",
+    "mobilePhone": "555-415-1337"
+  },
+  "credentials": {
+    "password": {},
+    "provider": {
+      "type": "IMPORT",
+      "name": "IMPORT"
+    }
+  },
+  "_links": {
+    "activate": {
+      "href": "https://${yourOktaDomain}/api/v1/users/00ub0oNGTSWTBKOLGLNR/lifecycle/activate"
+    },
+    "self": {
+      "href": "https://${yourOktaDomain}/api/v1/users/00ub0oNGTSWTBKOLGLNR"
+    }
+  }
+}
+```
+
+#### Create User with Password Import Inline Hook
+
+
+Creates a user with a specified [password hook](#password-hook-object). See [Password Import Inline Hook](/docs/reference/password-hook/) for more details.
+The new user is able to login immediately after activation with the specified password.
+This flow is common when migrating users from another data store in cases where we wish to allow the users to retain their current passwords.
+
+> Important: Do not generate or send a one-time activation token when activating users with an password hook.  Users should login with their existing password to be imported using the password import inline hook.
+
+##### Request Example
+
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+  "profile": {
+    "firstName": "Isaac",
+    "lastName": "Brock",
+    "email": "isaac.brock@example.com",
+    "login": "isaac.brock@example.com",
+    "mobilePhone": "555-415-1337"
+  },
+  "credentials": {
+    "password" : {
+      "hook": {
+        "type": "default"
+      }
+    }
+  }
+}' "https://${yourOktaDomain}/api/v1/users?activate=false"
+```
+
```

Comment:
Is there a need to also supply a recovery question?

Reply:
Not really! We usually try to keep such request examples minimal with emphasis on the example of the specific usage - `hook` in this example. 
Also, if you look at the example above for the `hash` based import, it also doesn't include `recovery question` etc. So, for purposes of consistency, I kept this as-is. 
Overall, this whole PR's examples are basically similar to import using the `Password Hash Object` and so the content and examples are consistent with that.